kuzzleio/documentation-V2
# Initialize Guide documentation for Kuzzle v2

:warning: Depends on #42 

## What does this PR do?

Copy the Kuzzle v1 Guide from the [documentation](https://github.com/kuzzleio/documentation) and adapt it to the new documentation system.

Also, h1 titles are now added to the table of content (previously only h2 titles were eligible). This makes H1 titles act as a make-shift "go to the top" link.

### How should this be manually tested?

Obviously, do NOT read the diff changes: there is almost no change to documentation itself!

Also, netlify won't show the new guide, because it has been added to the exclusion list (see #41). 

If you really want to double check the changes introduced by this PR, you'll have to pull the branch, and run the documentation locally, using development mode.
